openqa-label-known-issues

```diff
@@ -2,6 +2,7 @@
 
 # Usage
 # openqa-label-known-issues https://openqahost/tests/1234
+# Requires: perl-Text-Markdown html-xml-utils
```

Comment:
those are actually mentioned in 
https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml
So you only need to install  os-autoinst-scripts-deps

Reply:
Can at least update the readme? there is not any relevant info

Reply:
Mentioning this in the README seems like a great idea to me

Reply:
https://github.com/os-autoinst/scripts/commit/2b4f548c2e5b64d7f3059738f046fc204a379d0f